kolibri2zim/scraper.py

```diff
@@ -282,11 +273,12 @@ def funnel_from_s3(self, file_id, path, checksum, preset):
 
         # add to zim
         with self.creator_lock:
-            self.creator.add_item_for(
-                path=path,
-                content=fileobj.read(),
-                mimetype=preset.mimetype,
-            )
+            kwargs = {
+                "path": path,
+                "fileobj": fileobj,
+                "mimetype": preset.mimetype,
+            }
+            self.creator.add_item_for(StaticItem(**kwargs))
```

Comment:
No, this won't work. If you want to use an Item, use `add_item` but `add_item_for` doesn't accept it.
Why would you need that anyway? I don't understand

Reply:
Using `add_item_for` is an error from my side. I should have double-checked.

The rest consists simply in moving the `kwargs` outside of the argument, so that the type checker does not complains anymore (I don't know why) .
And I don't know why Github render the diff like this ...

Compared to `main`, the diff is now:
```diff
         # add to zim
         with self.creator_lock:
-            self.creator.add_item(
-                StaticItem(
-                    path=path,
-                    fileobj=fileobj,
-                    mimetype=preset.mimetype,
-                )
-            )
+            kwargs = {
+                "path": path,
+                "fileobj": fileobj,
+                "mimetype": preset.mimetype,
+            }
+            self.creator.add_item(StaticItem(**kwargs))
         logger.debug(f"Added {path} from S3::{key}")
```

Reply:
Nope, it is not sufficient ... Probably 3.1.2 once https://github.com/openzim/python-scraperlib/issues/109 is fixed.